### ISSUE TITLE
Header logo skewing in Safari when scrolling down on page

### ISSUE THREAD

**Issue by xpurichan** [OpenSourceOrg/dotOrg#111]
### Description

While testing an unrelated issue, we noticed that the header logo becomes squished when we scroll down on the website's pages (initial load is fine). This behavior does not occur in Chrome or Chrome in our tests, but is present in Safari (both mobile and desktop) and possibly other browsers too.

**Mobile:**
![IMG_1863](https://github.com/user-attachments/assets/778692b3-5786-40ea-92f0-3a57083ec79c)

**Desktop:**
![Screenshot 2024-08-19 at 11 19 16 AM](https://github.com/user-attachments/assets/32cdc47e-0f2d-4637-aa9d-c6f455e27965)

Possibly related to https://github.com/OpenSourceOrg/dotOrg/issues/104 cc @geoffguillain and @toncijajic

**Comment by geoffguillain** [comment]
Hey @xpurichan,

Thanks for finding this one out.
Safari seems to behave differently. I have updated the CSS to fix this issue.

I have tested on Chrome, FF, Safari (macOS + iOS)
**demo**: https://open-source-initiative-development.mystagingwebsite.com/

**Extra:**

There is some issue with the padding/margin on the hero banner on small devices. The words(even short ones) are clipping.

![image](https://github.com/user-attachments/assets/270111b9-687b-4f9e-8e8a-824b404e98fb)

**Comment by xpurichan** [comment]
Good catch @geoffguillain, I've copied their production banner on to the [[staging site](https://open-source-initiative-development.mystagingwebsite.com/)] and pitched this and the logo update to them. 

Awaiting the partner's approval now. I'll ping you on the logo update once approved.

**Comment by geoffguillain** [comment]
Thanks for letting me know @xpurichan 
Waiting for approval to merge on prod!

**Comment by nate-allen** [comment]
@xpurichan I'll be taking this over. Let me know when this is approved for production. Thank you

**Comment by moyvincent** [comment]
Thanks @xpurichan!
I have sent an update to the partner to review the [staging site](https://open-source-initiative-development.mystagingwebsite.com/) for the fix.

I will update back here when this has been approved for production.

### Reference
[Duplicate Issue](https://github.com/OpenSourceOrg/dotOrg/issues/165)

**Comment by ajolo2ky** [comment]
Hi team, sorry I couldn't find this issue. I validated this is working in staging, please promote it to production.

**Comment by moyvincent** [comment]
Hi @nate-allen 👋 , Please can we move the fix on the staging to production?

Thank you!

**Comment by nate-allen** [comment]
Pushed a fix to production and tested by scrolling on Safari

**Desktop**
![Image](https://github.com/user-attachments/assets/5f5231d4-c735-4c61-ba0e-913b7bdbfe59)

**Mobile**
![Image](https://github.com/user-attachments/assets/fc62713c-1ff5-431c-a794-55a3b6d9dbb8)

**Comment by xpurichan** [comment]
@nate-allen sorry, I am seeing an outstanding issue with the logo on mobile. When at the top of the page it can be too large, overlapping with the search icon:

![Image](https://github.com/user-attachments/assets/3635aa74-cd8e-4dc1-ae34-f59640cf16c1)

This specific example was taken from:
https://opensource.org/keep-up-with-open-source
It is present across different mobile browsers. I tested it with Chrome and Safari on an iPhone 13 Pro.

**Comment by nate-allen** [comment]
Taking another look now

**Comment by nate-allen** [comment]
@xpurichan I added a max-width to the logo so now it doesn't overlap and matches what you see on staging.

Tested on iPhone in Safari:

![Image](https://github.com/user-attachments/assets/8501c9e4-e45d-4d34-b080-83b9b2e4eb8b)

**Comment by moyvincent** [comment]
Thank you @nate-allen and @xpurichan!

I have updated the partner in Zendesk about this and will standby for feedback.

**Comment by ajolo2ky** [comment]
Confirming it's working on iOS (Safari/Chrome), Android (Chrome) and Safari/Mac.

Thanks!
